### PR TITLE
DON-15 DON-36 DON-37 campaign page

### PR DESCRIPTION
**Pull request by tbg-jasonfung**
- DON-15 DON-36 DON-37 campaign page

### CODE REVIEW

File: src/app/donation-start/donation-start.component.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
-<h1>Donating to campaign ID {{ campaignId }}!</h1>
-
-<form (ngSubmit)="submit()" [formGroup]="donationForm">
+<form class="formPadding" (ngSubmit)="submit()" [formGroup]="donationForm">
```
Comment (NoelLH) [inline src/app/donation-start/donation-start.component.html]: If the component only has one `form`, why not just target `form` in the SCSS? Angular will isolate styling's scope to the component for you.

File: src/app/campaign-details/campaign-details.component.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <div>
+        <h3>Campaign Targets</h3>
+        <p *ngIf="campaign.target > 0">Campaign Target: {{ campaign.target | currency : '£' : 'symbol' : '1.0-0' }}</p>
+        <p *ngIf="campaign.amountRaised > 0">Amount Raised: {{ campaign.amountRaised | currency : '£' : 'symbol' : '1.0-0' }}</p>
+    </div>
+
+    <div>
+        <button mat-raised-button color="primary" [routerLink]="'/donate/' + campaign.id">Donate</button>
+    </div>
+</div>
```
Comment (NoelLH) [inline src/app/campaign-details/campaign-details.component.html]: It's normal practice to have 1 newline at the end of files to keep diffs cleaner while following OS conventions - this is why GitHub adds this marker if it's missing

**Comment by tbg-jasonfung** [comment]
@NoelLH Thanks for the comments, i've resolved all of these now - let me know if theirs additional changes to make

**Comment by NoelLH** [comment]
> @NoelLH Thanks for the comments, i've resolved all of these now - let me know if theirs additional changes to make

Thanks @tbg-jasonfung, looks good! I've just added some minor style fixes on the same branch and will merge when the build passes.

**Pull request merged**